Apps/JobScheduler/src/Properties/launchSettings.json

```diff
@@ -4,7 +4,7 @@
     "windowsAuthentication": false,
     "anonymousAuthentication": true,
     "iisExpress": {
-      "applicationUrl": "http://localhost:5005"
+      "applicationUrl": "https://localhost:5005"
```

Comment:
I dont think this will work on localhost since https requires a certificate

Reply:
i had setup   local cert when debuggin issues on Friday, it hinted about cookies not exhanged over HTTPS but in the end it was the order of setting up cors and auth for Hangfire that caused the grief. I have since set it back